### ISSUE TITLE
Add LICENSE

### ISSUE THREAD

**Issue by Swiss-Mac-User** [cweagans/dotfiles#11]
I would like to incorporate some commands into [Swiss-Mac-User/macOS-scripted-setup](https://github.com/Swiss-Mac-User/macOS-scripted-setup) (with [attribution](https://github.com/Swiss-Mac-User/macOS-scripted-setup#inspiration-and-kudos) of course), but for this I would appreciate if you could state the license applicable for your repository.

**Comment by cweagans** [comment]
MIT licensed. Will add license file, but you can consider this the license in the interim if you'd like.

**Comment by cweagans** [comment]
Added a license. Did you ever end up doing anything with this?

**Comment by Swiss-Mac-User** [comment]
> Added a license. Did you ever end up doing anything with this?

Yes -> [Swiss-Mac-User/macOS-scripted-setup](https://github.com/Swiss-Mac-User/macOS-scripted-setup) 😎
